wadsrc/static/zscript/actors/inventory/inventory.zs

```diff
@@ -914,6 +914,25 @@ class Inventory : Actor
 		}
 	}
 
+	//===========================================================================
+	//
+	// Inventory :: DepleteBy
+	//
+	// Handles item depletion when using or taking items
+	//
+	//===========================================================================
+	virtual void DepleteBy(int by)
+	{
+		if (!amount || by >= amount)
```

Comment:
The check here should be `amount < 1`, not `!amount`

Reply:
Both changes pushed now.